mantle/cmd/kola/testiso.go

```diff
@@ -104,6 +106,31 @@ RequiredBy=multi-user.target
 RequiredBy=coreos-installer.target
 `, liveOKSignal)
 
+// TODO: replace unit with environment variables when available
+var preshimUnit = `[Unit]
+Before=coreos-installer.service
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c '/usr/bin/cp /usr/libexec/coreos-installer-service /usr/local/bin/'
+ExecStart=/bin/sh -c "/usr/bin/sed -i -r  's,coreos-installer (.*),coreos-installer \1 --offline,g' /usr/local/bin/coreos-installer-service"
+ExecStart=/bin/sh -c '/usr/bin/mount --bind /usr/local/bin/coreos-installer-service /usr/libexec/coreos-installer-service'
+[Install]
+RequiredBy=multi-user.target
+RequiredBy=coreos-installer.target
+`
+var downloadCheck = `[Unit]
+After=coreos-installer.service
+Before=coreos-installer.target
+Before=coreos-installer-reboot.service
```

Comment:
Could use a comment saying that this can be dropped with coreos-installer 0.5.1.